tests/README.md

```diff
@@ -6,3 +6,26 @@ From top-level directory run:
     $ tox
 
 **NOTE**: This requires `tox` ( please install with `pip install tox` ) and several supported versions of Python.
+
+Integration tests
+==========
+
+To run all of the integration test `modes` uncomment the following line from `tox.ini` and add the paths to your Kafka and Confluent Schema Registry instances. 
+You can also run the integration tests outside of `tox` by running this command from the source root.
+
+    examples/integration_test.py <kafka-broker> [<test-topic>] [<schema-registry>]
+
+To run individual integration test `modes` with `tox` use the following syntax.
+
+    tox -- --<test mode>
+
+The throttle_cb integration test require an additiona step and as such are not included in the default test mode.
+In order to execute the throttle_cb test you must first set a throttle for the client 'throttled_client' with the command below:
+
+    kafka-configs  --zookeeper <zookeeper host>:<zookeeper port> \
+        --alter --add-config 'request_percentage=01' \
+        --entity-name throttled_client --entity-type clients
+
```

Comment:
out of curio, is it possible to do this with alterConfigs now?

Reply:
possibly, worth a try I suppose. I'll push the other review changes first then revisit

Reply:
Not until 2.0 it would seem 

https://cwiki.apache.org/confluence/display/KAFKA/KIP-257+-+Configurable+Quota+Management